charts/dex/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 type: application
 name: dex
-version: 0.11.1
-appVersion: "2.34.0"
+version: 0.12.0
+appVersion: "2.35.0"
```

Comment:
Can you please change this to 2.35.1? We've tagged a new release fixing a regression.

Reply:
Sure!